tests/wsl/install_wsl.pm

```diff
@@ -0,0 +1,128 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Install SUSE or openSUSE WSL images from the MS Store directly
+# Maintainer: qa-c  <qa-c@suse.de>
+
+use Mojo::Base "windowsbasetest";
+use testapi;
+use version_utils;
+use Utils::Architectures 'is_aarch64';
+
+sub install_certificates {
+    my ($self) = @_;
+    if (is_sle('>=15-sp2') || is_opensuse) {
```

Comment:
Is the other code path still relevant? If not, it could be removed IMO

Reply:
Yes, as 12-SP5 is LTSS in 31th Oct of 2027, we've decided to leave this part just in case there's a new 12-SP5 build.

Reply:
Yes but, hasn't the certificate stopped being inside the `appx` in newer versions? That's why I thought this code was there... :thinking:

Reply:
No, it always has to be there AFAIK

Reply:
Maybe this should be refactored in a future PR

Reply:
Testing with needles in Windows is always a source of pain and problems, so the "extracting method" is not recommended unless 12-SP5 gets some new build.

Reply:
The PS method should work on 12 SP5 as well.

Reply:
We finally decided to remove the 12-SP5 path